views/lexicon/forms/checkbox.html.twig

```diff
@@ -67,7 +67,6 @@
         form.checkbox_inline({
             'id': 'checkbox-7',
             'label': 'Checkbox inline plus extras',
-            'help': 'Help text to give more information about expected input',
```

Comment:
Why was this removed?

Reply:
![help-w3cerror](https://user-images.githubusercontent.com/1425876/30534559-a05fe6ce-9c56-11e7-83be-7c6b50e22a7b.png)

Reply:
This is the state of the element with the 'help' attribute:
![help2-w3cerror](https://user-images.githubusercontent.com/1425876/30534684-2f06de50-9c57-11e7-9d80-5c52bf35e382.png)

Reply:
This is the state of the element without the 'help' attribute:
![Uploading help3-W3CError.png…]()

No visual regression